src/cirrus/lib/statedb.py

```diff
@@ -25,6 +25,12 @@ def get_state_db():
     return os.getenv("CIRRUS_STATE_DB")
 
 
+def to_current(item: dict[str, Any]) -> dict[str, Any]:
+    item["catid"] = item["payload_id"]
```

Comment:
I'm not sure how to best describe this function, but my thought is something like below, IIUC.
```suggestion
    """Compatiblity function for cirrus-dashboard""""
    item["catid"] = item["payload_id"]
```

Reply:
I like. Better than naught. Inserted.